### PR TITLE
Add date time stamps to email

### PR DESCRIPTION
**Pull request by hllavina**
Creates date time string in process.php with other opt-out custom
handling

### CODE REVIEW

File: scripts/php/process.php [hunk 1]
```diff
@@ -169,6 +169,7 @@ private function buildMessageText () {
       $this->messageText = preg_replace("/\[>optOutHello<\]/U", $this->buildOptOutHello(), $this->messageText);
       $this->messageText = preg_replace("/\[>optOutRequestor<\]/U", $this->buildOptOutRequestor(), $this->messageText);
       $this->messageText = preg_replace("/\[>optOutCoauthorLabel<\]/U", $this->buildOptOutCoauthorLabel(), $this->messageText);
+     $this->messageText = preg_replace("/\[>optOutDateTimeStamp<\]/U", $this->buildOptOutDateTimeString(), $this->messageText);
```
Comment (frrrances) [inline scripts/php/process.php]: could this be just a generic timestamp we could use in any email?

File: scripts/php/process.php [hunk 2]
```diff
@@ -211,6 +212,12 @@ private function buildOptOutRequestor () {
     return $requestor;
   }
 
+  private function buildOptOutDateTimeString () {
```
Comment (frrrances) [inline scripts/php/process.php]: could this be just a generic function we could use to add a timestamp to any email?

File: forms-text/opt-out.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
+
+This MIT Authors' Opt-In Open Access License was signed on: [>optOutDateTimeStamp<].
```
Comment (frrrances) [inline forms-text/opt-out.txt]: I think this should be "Opt-Out Open Access License" instead of "Opt-In" since this is the email for opting out.

**Review by frrrances — COMMENTED** [review]
A couple questions, and also I just realized that the text should go in the opt-in.txt file as well.

**Review by frrrances — APPROVED** [review]
Looks good! 👍

**Review by matt-bernhardt — APPROVED** [review]
Looks good

**Pull request merged**